pom.xml

```diff
@@ -21,8 +21,8 @@
     <scm>
         <url>https://github.com/vitalidze/chromecast-java-api-v2</url>
         <connection>scm:git:https://github.com/vitalidze/chromecast-java-api-v2.git</connection>
-      <tag>HEAD</tag>
-  </scm>
+        <tag>HEAD</tag>
```

Comment:
Please revert this. You are welcome to submit a separate pull request with indentation changes if needed.